main/samba/samba.initd

```diff
@@ -1,24 +1,41 @@
 #!/sbin/openrc-run
 
 extra_started_commands="reload"
+piddir=${piddir:-"/run/samba"}
+daemon_list="${daemon_list//samba/}"
```

Comment:
Why are we always removing `samba` daemon?

It would be nice to avoid `${var//search/replace}` expansion if possible, since it is not specified in POSIX shell.

Reply:
As far as I know, samba(4) is not compatible with smbd, nmbd. It is compatible with winbindd for purpose but samba launches winbindd itself. So, I'm trying to solve the any possible conflict between old (smbd, nmbd, winbindd) and new (samba). For instance, Debian's default install comes with samba-ad-dc service masked. You have to mask nmbd, smbd, winbindd and then unmask samba-ad-dc to use samba's ad-dc functionality in order to solve the conflicts.